$DragonFly: src/gnu/usr.bin/sdiff/sdiff.c.patch,v 1.1 2004/04/13 14:55:02 eirikn Exp $
--- sdiff.c.orig	Fri Mar  5 14:18:20 2004
+++ sdiff.c	Fri Mar  5 14:18:35 2004
@@ -453,7 +453,6 @@
   initialize_main (&argc, &argv);
   program_name = argv[0];
   setlocale (LC_ALL, "");
-  bindtextdomain (PACKAGE, LOCALEDIR);
   textdomain (PACKAGE);
   c_stack_action (c_stack_die);
 
